EIPS/eip-4804.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 4804
+title: Exodus
+description: Exodus allows to create blockchain networks with accounts and transactions.
+editors: Antoine Toulmé (@atoulme), Chaals Nevile (@chaals)
```

Comment:
```suggestion
author: Antoine Toulmé (@atoulme), Chaals Nevile (@chaals)
```
See https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for a template.